doc/releases/changelog-0.37.0.md

```diff
@@ -167,35 +167,135 @@ via the `NoiseModel` class and an `add_noise` transform.
   To learn more about this new functionality, check out our [noise module documentation](https://docs.pennylane.ai/en/stable/code/qml_noise.html)
   and keep your eyes peeled for an in-depth demo!
 
-<h4>Identify mistakes in your code with the PennyLane debugger 🚫🐞</h4>
+<h4>Catch bugs with the PennyLane debugger 🚫🐞</h4>
 
-* Added a quantum debugger (`PLDB`) which interfaces via `qml.breakpoint()` and provides tools for 
-  debugging quantum circuits. Users can step through the quantum circuit operations, dynamically
-  queue operations and make measurements using (`qml.debug_state()`, `qml.debug_probs()`, 
-  `qml.debug_expval()`, and `qml.debug_tape()`).
+* The new PennyLane quantum debugger (`PLDB`) allows pausing simulation via the `qml.breakpoint()` command and provides tools for 
```

Comment:
How much do we want to market PLDB as a thing? I'm not sure we need to so much :thinking:

Reply:
This is a good question. It's kinda making it look like its own standalone product if we give it an acronym 😅. But, I see the use for it because the UI has `pldb` all over the place. @DSGuala what about if we just make it lower-case? 🤔 

```suggestion
* The new PennyLane quantum debugger (`pldb`) allows pausing simulation via the `qml.breakpoint()` command and provides tools for 
```

Reply:
Decided to just remove it.

Reply:
I still see it :thinking: not a blocker though.